src/jit/codegencommon.cpp

```diff
@@ -2255,11 +2255,14 @@ void CodeGen::genGenerateCode(void** codePtr, ULONG* nativeSizeOfCode)
                 printf("X64 CPU with SSE2");
             }
         }
-
         else if (compiler->info.genCPU == CPU_ARM)
         {
             printf("generic ARM CPU");
         }
+        else if (compiler->info.genCPU == CPU_ARM64)
+        {
+            printf("generic ARM64 CPU");
+        }
```

Comment:
Should there be an else assert here?

Reply:
We shouldn't have asserts in the output code, although printing "Unknown" makes sense.

---
In reply to: [223864659](https://github.com/dotnet/coreclr/pull/20327#discussion_r223864659) [](ancestors = 223864659)